### PR TITLE
Add temurin update script

### PR DESCRIPTION
**Pull request by loosebazooka**
This needs to be integrated as an action, but should run daily so we can pick up the latest temurin builds for java21 at the moment and other java builds in the future

It was an interesting use of a day.

### CODE REVIEW

**Comment by loosebazooka** [comment]
@thesayyn perhaps there is a more bazel-y way to do this? But this covers the process.

File: update_java_archives.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+# Copyright 2024 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (thesayyn) [inline update_java_archives.sh:6]: Can this go into the knife script?

**Comment by thesayyn** [comment]
Also this could definitely be a bazel script instead...

**Review by thesayyn — APPROVED** [review]
Sweet

**Pull request merged**